### PR TITLE
New node x position & well applied y axis position

### PR DESCRIPTION
**Pull request by Delfil**
Position on the x axis of a node is now determined by the previous node count instead of the previous base pair count.
The semantic zooming is now determined by the space left between nodes.

### CODE REVIEW

**Comment by borismattijssen** [comment]
I think its still not a very clear overview. But I think it's good enough for the demo

**Comment by Delfil** [comment]
There are still some tests missing and the spacing isn't optimised yet, but the future is fully implemented, and it is useful to work from on other branches & it is an better situation then the current x and y positioning.

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
 	 * The space left between nodes for drawing the edges between the nodes.<br>
 	 * Set the zoom to have changes take effect.
 	 */
-//	private int nodeJumpSize = 20;
+	// private int nodeJumpSize = 20;
```
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: Will this ever be used again actually?

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java [hunk 2]
```diff
@@ -73,9 +73,10 @@ public void init() {
 	 * zoomLevel updates.
 	 */
 	private void generateViewer() {
-		viewer = new Viewer(graph, Viewer.ThreadingModel.GRAPH_IN_ANOTHER_THREAD);
+		viewer = new Viewer(graph,
+				Viewer.ThreadingModel.GRAPH_IN_ANOTHER_THREAD);
 		panel = viewer.addDefaultView(false);
-//		panel.getCamera().setAutoFitView(false);
+		// panel.getCamera().setAutoFitView(false);
```
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: Can be removed too I guess?

**Comment by mathieupost** [comment]
Rest looks good!

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java (hunk 2 above)
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: Idd, removing it now.

File: src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java (hunk 1 above)
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/visualization/GraphView.java]: Not anymore, removing it now.

**Pull request merged**